contrib/coredns/policy/policy.go

```diff
@@ -51,9 +55,16 @@ type PolicyMiddleware struct {
 	ErrorFunc func(dns.ResponseWriter, *dns.Msg, int) // failover error handler
 }
 
+type Attribute struct {
+	Id    string
+	Type  string
+	Value string
+}
+
 type Response struct {
-	Permit   bool   `pdp:"Effect"`
-	Redirect net.IP `pdp:"redirect_to"`
+	Permit     bool   `pdp:"Effect"`
+	Redirect   net.IP `pdp:"redirect_to"`
+	Obligation []*Attribute
```

Comment:
Hello, my understanding is that these changes are checked in, I could not find policy_id in PDP server. In service.pb.response, Response structure is 

type Response struct {
	Effect     Response_Effect `protobuf:"varint,1,opt,name=effect,enum=service.Response_Effect" json:"effect,omitempty"`
	Reason     string          `protobuf:"bytes,2,opt,name=reason" json:"reason,omitempty"`
	Obligation []*Attribute    `protobuf:"bytes,3,rep,name=obligation" json:"obligation,omitempty"`
}

We make the validate call 

func (c *pDPClient) Validate(ctx context.Context, in *Request, opts ...grpc.CallOption) (*Response, error) {
	out := new(Response)
	err := grpc.Invoke(ctx, "/service.PDP/Validate", in, out, c.cc, opts...)

Returned structure is Response above.

I am making the change assuming that fillResponse in client will return PolicyId